cmd/session.go

```diff
@@ -354,3 +327,69 @@ func SessionTerminate(cmd *cobra.Command, args []string) error {
 	ui.Successf("Session terminated")
 	return nil
 }
+
+// sessionSelectSSHExecRef selects an exec id from all sessions in the Unweave environment or whether to create a new
+// provides an option to create a new Exec an error or exits if unrecoverable
+func sessionSelectSSHExecRef(cmd *cobra.Command, execRef string, allowNew bool) (string, bool, error) {
+	const newSessionOpt = "create a new session"
```

Comment:
Could we call this `✨ Create a new session` so that it catches the user's eye as something different to the other options

Reply:
Implemented